automation/taskcluster/decision_task_release.py

```diff
@@ -0,0 +1,209 @@
+    owner="skaspari@mozilla.com",
+    source="https://github.com/mozilla-mobile/android-components.git"
+)
+
+
+def load_artifacts_manifest():
+    return yaml.load(open('automation/taskcluster/artifacts.yml', 'r'))
+
+
+def fetch_build_task_artifacts():
+    artifacts_info = load_artifacts_manifest()
+    artifacts = {}
+    for artifact, info in artifacts_info.items():
+        artifacts[artifact] = {
+            'type': 'file',
+            'expires': taskcluster.stringDate(taskcluster.fromNow('1 year')),
+            'path': info['path']
+        }
+
+    return artifacts
+
+
+def generate_build_task(version, _release):
+    # TODO: to uncomment when ready
+    # checkout = ("git fetch origin --tags && "
+                # "git config advice.detachedHead false && "
+                # "git checkout %s" % (version))
+    checkout = ("git fetch https://github.com/MihaiTabara/android-components.git bug1491314 && "
+                "git config advice.detachedHead false && "
+                "git checkout {}".format(version))
+
+    assemble_task = 'assembleRelease'
+    artifacts = fetch_build_task_artifacts()
+
+    return taskcluster.slugId(), BUILDER.build_task(
+        name="Android Components - Release ({})".format(_release),
+        description="Building and publishing release versions.",
+        command=(checkout +
+                 ' && ./gradlew --no-daemon clean test detektCheck ktlint '
+                 + assemble_task +
+                 ' docs lint uploadArchives zipMavenArtifacts'),  # TODO: add the bintray api-key uploading too
+        features={
+            "chainOfTrust": True
+        },
+        worker_type='gecko-focus',
+        scopes=[],  # TODO: to add "secrets:get:project/android-components/publish" when ready to switch back
+        artifacts=artifacts,
+    )
+
+
+def generate_massager_task(build_task_id, massager_task_id,
+                           artifacts_info, version):
+    upstreamZip = []
+    for artifact, info in artifacts_info.items():
+        new_artifact_id = None if info['name'] == info['old_artifact_id'] else info['name']
+        new_group_id = info['new_group_id'] if new_artifact_id else None
+        upstreamZip.append({
+                "path": artifact,
+                "taskId": build_task_id,
+                "newGroupId": new_group_id,
+                "newArtifactId": new_artifact_id,
+        })
+    scopes = []
+    # TODO: fix the command by using tags as in `build task
+    command = ("git fetch https://github.com/MihaiTabara/android-components.git bug1491314 && "
+               "git config advice.detachedHead false && "
+               "git checkout {} && "
+               "apt-get install -y python3-pip && "
+               "pip3 install scriptworker && "
+               "python3 automation/taskcluster/release/convert_group_and_artifact_ids.py {}".format(version, massager_task_id))
+
+    return BUILDER.massager_task(
+        name="Android Components - Massager",
```

Comment:
I love the name. It's really releng! If it wasn't a temporary solution, I would have suggested a more specific one.